File: Concurrent.java
public class Concurrent extends Thread{

    public static void pause(int time){
        try{
            Thread.sleep(time);
        }
        catch(InterruptedException e){
            Thread.currentThread().interrupt();
            System.err.format("IOException: %s%n", e);
        }

    }

    public static void byeWorld(){
        System.out.println("Bye World");
    }
    public static void main(String[] args){
        Concurrent thread = new Concurrent();
        
        System.out.println("Hello World");
        thread.start();
        pause(5000);
        

        
        
    }

    public void run(){
        System.out.println("Bye World");
    }
}

File: Concurrent.py
import time
import threading
from threading import Thread

def threadfunction():
    print("Hello World")
    time.sleep(5)
    print("Bye World")

def threadfunction2():
    print("Hello World 2")
    print("Bye World 2")

if __name__ == "__main__":
    x = Thread(target=threadfunction)
    y = Thread(target=threadfunction2)
    x.start()
    y.start()
    x.join()
    y.join()
